packages/wonder-blocks-tooltip/src/components/__tests__/tooltip.test.js

```diff
@@ -399,4 +399,54 @@ describe("Tooltip", () => {
             });
         });
     });
+
+    describe("Controlled", () => {
```

Comment:
I can convert these tests cases to use `@testing-library` whenever I get around to landing my PR.